packages/blocks/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 6.0.5 (Unreleased)
+
+### New features
+
+- `innerBlocks` is no available as a second attribute in the transform function
```

Comment:
Typo: "no" should be "now". Makes it sound as the opposite of an introduced new feature 😅